geode-docs/managing/statistics/how_statistics_work.html.md.erb

```diff
@@ -19,10 +19,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-Each application or cache server that joins the distributed system can collect and archive statistical data for analyzing system performance.
+<%=vars.product_name%> statistics can be enabled at a cluster level, an application or cache server level, or a region level to collect and archive statistical data for analyzing system performance.
+Statistics gathered at the cluster and application or cache server level are saved to a file and can be archived, whereas region level statistics are transient and accessible only through the API.
```

Comment:
This is were I find the term "region level statistics" extra confusing. Instead of talking about all the different levels perhaps you could just include a note that this section does not apply to the statistics described by the CacheStatistics interface and enabled with the region's statistics-enabled attribute which are transient and accessible only through the API.

Reply:
I agree that the introduction of this term is not the right way to go. Let's see if we can rewrite this to avoid talking about region-level stats or cluster/cache stats as terminology.